src/main/java/org/jitsi/impl/protocol/xmpp/ChatRoomImpl.java

```diff
@@ -844,6 +852,51 @@ public void processPresence(Presence presence)
         }
     }
 
+    /**
+     * {@inheritDoc}
+     */
+    public boolean isAvModerationEnabled()
+    {
+        return avModerationEnabled;
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    public void setAvModerationEnabled(boolean value)
+    {
+        this.avModerationEnabled = value;
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    public void updateAvModerationWhitelists(Map<String, List<String>> whitelists)
+    {
+        this.whitelists = whitelists;
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    public boolean isMemberAllowedToUnmute(Jid jid, MediaType mediaType)
+    {
+        if (!this.isAvModerationEnabled())
+        {
+            return true;
+        }
+
+        // there is no whitelists received yet
+        if (this.whitelists == null)
+        {
+            return false;
+        }
+
+        List<String> whitelist = this.whitelists.get(mediaType.toString());
+
+        return whitelist == null ? false : this.whitelists.get(mediaType.toString()).contains(jid.toString());
```

Comment:
`this.whitelists.get(mediaType.toString())` is already saved in the local `whitelist`.